scripts/core_tests_list.json

```diff
@@ -1,71 +1,3 @@
 [
-    "test_all_rules",
-    "test_catalog",
-    "test_collection_mtime",
-    "test_control_plane",
-    "test_delay_queue",
-    "test_dynamic_peps",
-    "test_iadmin",
-    "test_ibun",
-    "test_ichksum",
-    "test_ichmod",
-    "test_icommands_file_operations",
-    "test_icp",
-    "test_ifsck",
-    "test_ils",
-    "test_imeta_error_handling",
-    "test_imeta_help",
-    "test_imeta_interactive",
-    "test_imeta_set",
-    "test_imv",
-    "test_ipasswd",
-    "test_iphymv",
-    "test_iput_options",
-    "test_ipwd",
-    "test_iquest",
-    "test_ireg",
-    "test_irepl",
-    "test_irm",
-    "test_irmdir",
-    "test_irmtrash",
-    "test_irodsctl",
-    "test_irsync",
-    "test_irule",
-    "test_iscan",
-    "test_istream",
-    "test_iticket",
-    "test_itouch",
-    "test_itrim",
-    "test_iunreg",
-    "test_iuserinfo",
-    "test_izonereport",
-    "test_load_balanced_suite",
-    "test_misc",
-    "test_native_rule_engine_plugin",
-    "test_python_rule_engine_plugin",
-    "test_prep_genquery_iterator",
-    "test_quotas",
-    "test_resource_configuration",
-    "test_resource_types.Test_Resource_Compound",
-    "test_resource_types.Test_Resource_CompoundWithMockarchive",
-    "test_resource_types.Test_Resource_CompoundWithUnivmss",
-    "test_resource_types.Test_Resource_Deferred",
-    "test_resource_types.Test_Resource_MultiLayered",
-    "test_resource_types.Test_Resource_NonBlocking",
-    "test_resource_types.Test_Resource_Passthru",
-    "test_resource_types.Test_Resource_Random",
-    "test_resource_types.Test_Resource_RandomWithinRandom",
-    "test_resource_types.Test_Resource_RandomWithinReplication",
-    "test_resource_types.Test_Resource_Replication",
-    "test_resource_types.Test_Resource_ReplicationToTwoCompound",
-    "test_resource_types.Test_Resource_ReplicationToTwoCompoundResourcesWithPreferArchive",
-    "test_resource_types.Test_Resource_ReplicationWithinReplication",
-    "test_resource_types.Test_Resource_Unixfilesystem",
-    "test_resource_types.Test_Resource_WeightedPassthru",
-    "test_rule_engine_plugin_framework",
-    "test_rule_engine_plugin_passthrough",
-    "test_rulebase",
-    "test_ssl",
-    "test_stacktrace",
-    "test_symlink_operations"
+    "test_all_rules"
```

Comment:
is this correct?   removing all the others?

Reply:
I'm familiar with this trick :)

irods/irods_testing_jenkins#107